2_process.R

```diff
@@ -30,5 +30,15 @@ p2_targets_list <- list(
       create_site_list(wqp_data_subset,p1_nwis_sites,p1_daily_data,p1_inst_data,
                        hucs=drb_huc8s,crs_out="NAD83",fileout = "2_process/out/DRB_SC_sitelist.csv")
     },
-    format = "file")
+    format = "file"),
+  
+  # Aggregate instantaneous DO data to daily min/mean/max
+  tar_target(
+    p2_inst_data_daily,
+    aggregate_data_to_daily(p1_inst_data,p1_daily_data)),
+  
+  # Combine 1) daily DO data and 2) instantaneous DO data that has been aggregated to daily 
+  tar_target(
+    p2_daily_combined,
+    bind_rows(p1_daily_data, p2_inst_data_daily))
```

Comment:
Thanks for initializing this target that combines the NWIS continuous datasets. Do we still want to pad the timeseries to format for modeling (per discussion in #15)? If so, it might make sense to do that formatting within this target. What do you think?

Reply:
I think that the padding can happen downstream of this step. At least in past work it has happened downstream, closer to where the actual dynamic modeling happens.